openlibrary/core/models.py

```diff
@@ -1165,7 +1159,7 @@ def create(
                     'name': tag_name,
                     'tag_description': tag_description,
                     'tag_type': tag_type,
-                    'tag_plugins': json.loads(tag_plugins or "[]"),
+                    'tag_plugins': tag_plugins or "[]",
```

Comment:
We either want or an empty list, not as string of an empty list.
```suggestion
                    'tag_plugins': tag_plugins or [],
```